src/tricicl/loggers/tb.py

```diff
@@ -0,0 +1,44 @@
+from avalanche.evaluation.metric_results import AlternativeValues, MetricValue
+from avalanche.logging import StrategyLogger
+from matplotlib.figure import Figure
+from PIL.Image import Image
+from torch import Tensor
+from torch.utils.tensorboard import SummaryWriter
+from torchvision.transforms.functional import to_tensor
+
+from tricicl.utils.path import safe_path
+
+
+class TensorboardLogger(StrategyLogger):
```

Comment:
This logger should work as well for AWS S3, by simply giving it the s3 path, but I never tested Tensorboard on it